recipes/progress_reporter/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "progress_reporter" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }} 
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/marscher/progress_reporter
+  git_rev: {{ 'v' + version }}
+
+build:
+  noarch_python: True
+  number: 0
+# since this is noarch, only build on linux
+  skip: True  # [not linux]
```

Comment:
no worries, thank you very much for the references. I'm currently reading this code https://github.com/conda-forge/conda-forge-build-setup-feedstock/blob/master/recipe/upload_or_check_non_existence.py and actually it should notice if a noarch package does exist already.

But dont lets get too offtopic here now :)